electrum/lnwire/README.md

```diff
@@ -1,5 +1,23 @@
-These files are generated from the BOLT repository:
+These files have been generated from the BOLT repository:
 ```
 $ python3 tools/extract-formats.py 01-*.md 02-*.md 07-*.md  > peer_wire.csv
 $ python3 tools/extract-formats.py 04-*.md  > onion_wire.csv
 ```
+
+Notes:
+
+ Trampoline messages were added manually to onion_wire.csv
+
+ Some options were added manually to peer_wire.csv:
+
+47,148c147,148
+< msgdata,channel_reestablish,your_last_per_commitment_secret,byte,32,option_data_loss_protect,option_static_remotekey
+< msgdata,channel_reestablish,my_current_per_commitment_point,point,,option_data_loss_protect,option_static_remotekey
+---
+> msgdata,channel_reestablish,your_last_per_commitment_secret,byte,32
+> msgdata,channel_reestablish,my_current_per_commitment_point,point,
+188c188
+< msgdata,channel_update,htlc_maximum_msat,u64,,option_channel_htlc_max
+---
+> msgdata,channel_update,htlc_maximum_msat,u64,
+
```

Comment:
First of all this is a markdown file, so why not use the formatting capabilities and put this inside a code-block for readability? :P

Besides, I think the script output is correct and these manual modifications are not needed.

Re `msgdata,channel_update,htlc_maximum_msat`, that field used to be conditional based on `option_channel_htlc_max`, but https://github.com/lightning/bolts/pull/999 made it mandatory.
Re the `channel_reestablish` fields, similar thing: they used to be conditional on those options but https://github.com/lightning/bolts/commit/f068dd0d8dfa5ae75feedd99f269e23be4777381 made them mandatory.

Reply:
all right, I will remove those.